src/views/statutory/EditCandidate.vue

```diff
@@ -114,7 +114,7 @@
         </div>
 
         <div class="field">
-          <label class="label">Languages</label>
+          <label class="label">Languages<tooltip text="To add a language to the list, press either 'tab' or 'enter', or add a comma"/></label>
```

Comment:
can you move this to newline like this:

```
<label class="label">Languages
    <tooltip text="To add a language to the list, press either 'tab' or 'enter', or add a comma" />
</label>
```

so it'd look more nice?

Reply:
Sure